addons/l10n_cn/data/template/account.account-cn.csv

```diff
@@ -1,77 +1,38 @@
-"l10n_cn_6101","Gains and Losses of fair value change","6101","income_other","False","公允价值变动损益"
-"l10n_cn_6111","Income from investment","6111","income_other","False","投资收益"
-"l10n_cn_6301","Non-operating Income","6301","income_other","False","营业外收入"
-"l10n_cn_6401","Main Business Cost","6401","expense","False","主营营业成本"
-"l10n_cn_6402","Other Operating Costs","6402","expense","False","其他业务成本"
-"l10n_cn_6403","Operating Taxes and Surcharges","6403","expense","False","营业税及附加"
-"l10n_cn_6601","Selling Expenses","6601","expense","False","销售费用"
-"l10n_cn_6602","Management Expenses","6602","expense","False","管理费用"
-"l10n_cn_6603","Financial Expenses","6603","expense","False","财务费用"
-"l10n_cn_6701","Assets impairment Loss","6701","expense","False","资产减值损失"
-"l10n_cn_6711","Non-operating expenses","6711","expense","False","营业外支出"
-"l10n_cn_6801","Income Tax Expense","6801","expense","False","所得税费用"
-"l10n_cn_6901","Prior year income adjustment","6901","expense","False","以前年度损益调整"
+"l10n_cn_110100","Transactional Financial Assets","1101","asset_current","False","交易性金融资产"
+"l10n_cn_101200","Other Monetary Funds","1012","asset_current","False","其他货币资金"
+"l10n_cn_112100","Bills Receivable","1121","asset_receivable","True","应收票据"
+"l10n_cn_112300","Advance Payment","1123","asset_prepayments","False","预付账款"
+"l10n_cn_113100","Divident Receivable","1131","asset_receivable","True","应收股利"
+"l10n_cn_113200","Interest Receivable","1132","asset_receivable","True","应收利息"
+"l10n_cn_122100","Other Receivable","1221","asset_receivable","True","其他应收款"
+"l10n_cn_123100","Bad Debt Provisions","1231","asset_current","False","坏账准备"
+"l10n_cn_140100","Material Purchasing","1401","asset_current","False","材料采购"
+"l10n_cn_140600","Goods shipped in transit","1406","asset_current","False","运输中的货物"
+"l10n_cn_140700","Differences between purchasing and selling price","1407","asset_current","False","商品进销差价"
+"l10n_cn_140800","Consigned processing materials","1408","asset_current","False","委托加工的材料"
+"l10n_cn_147100","Inventory falling price reserves","1471","asset_current","False","存货跌价准备"
+"l10n_cn_153100","Long-term receivables","1531","asset_non_current","True","长期应收款"
+"l10n_cn_160200","Accumulated depreciation","1602","expense_depreciation","False","累计折旧"
+"l10n_cn_160300","Fixed assets depreciation reserves","1603","expense_depreciation","False","固定资产减值准备"
+"l10n_cn_160500","Engineering materials","1605","asset_non_current","False","工程物资"
+"l10n_cn_180100","Long-term amortized expenses","1801","expense_depreciation","False","长期待摊销费用"
+"l10n_cn_200100","Short-term borrowing","2001","liability_payable","True","短期借款"
+"l10n_cn_220100","Bills Payable","2201","liability_payable","True","应付票据"
+"l10n_cn_220300","Deposit Received","2203","liability_payable","True","预收账款"
+"l10n_cn_221100","Payroll payable","2211","liability_payable","True","应付职工薪酬"
+"l10n_cn_222100","Tax payable","2221","liability_current","True","应交税金"
```

Comment:
This account is there twice, once in the here and once in the large business file.
Shouldn't it appear only once?

I also noticed that account codes in this file are shorter than in the other two, it should be the same everywhere no? Especially since the common COA is installed too.

Also, I tested installing the normal COA (not large company) and it failed because the taxes couldn't find that account, most likely because it wasn't yet installed since the taxes are enabled with the common COA.

Reply:
Those fields have slightly changes compared with normal COA (Chinese/eng name diff, reconcile from true to false)
But I think I should differentiate them by different ids. Let me ping you again when I changed all the ids

And would fix the tax id also